packages/elements/src/components/ui/buttons/common/ButtonContent.tsx

```diff
@@ -35,13 +36,16 @@ export const ButtonContent: React.FC<ButtonContentProps> = ({
   spinnerClassName,
   leftWrapperClassName,
   rightWrapperClassName,
+  iconSize = "medium",
 }) => {
+  const iconPixelSize = iconSize === "small" ? 12 : undefined;
```

Comment:
We should also handle "large" here.

```suggestion
  const iconPixelSize = iconSize === "small" ? 12 : iconSize === "large" ? 24 : undefined;
```